chart/templates/_helpers.tpl

```diff
@@ -713,6 +823,28 @@ Create the name of the service account to use for the cdn-invalidator-sub deploy
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the proper image name for cdnInvalidatorSub
+*/}}
+{{- define "carto.cdnInvalidatorSub.image" -}}
+{{- if .Values.cdnInvalidatorSub.image.name -}}
+{{- .Values.cdnInvalidatorSub.image.name -}}
+{{- else -}}
+{{- printf "%s/%s" .Values.global.imageRegistry "cdn-invalidator-sub" -}}
```

Comment:
Ey, that should be:
```suggestion
{{- printf "%s/%s" .Values.global.imageRegistry "consumers/cdn-invalidator-sub" -}}
```

(According with the value that you deleted from `Values.yaml`)

Reply:
I overlooked it, thanks 🤗